### PR TITLE
chaos-daemon support arm64

### PR DESCRIPTION
**Pull request by iguoyr**
### What problem does this PR solve?

Problem Summary: 

Run chaos-mesh on arm64 machine, and we can pass `make check` and `make test` on arm64 machine now.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

File: images/chaos-daemon/Dockerfile [hunk 1]
```diff
@@ -6,23 +6,28 @@ ARG HTTP_PROXY
...
+    export NSEXEC_ARCH='x86_64'; \
+    ;; \
+    'arm64') \
+    export NSEXEC_ARCH='aarch64'; \
+    ;; \
+    *) echo >&2 "error: unsupported architecture '$TARGET_PLATFORM'"; exit 1 ;; \
+    esac; \
+    curl -L https://github.com/chaos-mesh/nsexec/releases/download/v0.1.6/nsexec-$NSEXEC_ARCH-unknown-linux-gnu.tar.gz | tar xz -C /usr/local/bin; \
+    cp /usr/local/bin/libnsenter.so /usr/local/lib/libnsenter.so
+RUN curl -L https://github.com/chaos-mesh/rs-tproxy/releases/download/v0.2.3/tproxy-linux-amd64.tar.gz | tar xz -C /usr/local/bin
```
Comment (YangKeao) [inline images/chaos-daemon/Dockerfile:30]: The newer version of `tproxy` also supports `aarch64`.

Comment (iguoyr) [reply images/chaos-daemon/Dockerfile]: updated!

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**